docs/README.md

```diff
@@ -1,62 +1,218 @@
+- "I don't know how to write."
+    
+    We prepared some [recommendations](#what-to-write) for you.
 
-# How to Write Content for ClickHouse Documentation
+- "I know what I want to write, but I don't know how to contribute to docs."
 
-## Target Audience
+    Here are some [tips](#how-to-contribute).
 
-When you write pretty much any text, the first thing you should think about is who will read it and which terms you should use for communicating with them.
+- "I don't know what to describe."
 
-ClickHouse can be directly used by all sorts of analysts and engineers. For generic parts of documentation (like the query language, tutorials or overviews), assume that the reader only has a basic technical background. For more technical sections (like articles that describe ClickHouse internals, guides for operating ClickHouse clusters, or rules for contributing to C++ code), you can use technical language and concepts.
+    Ask us, and we will provide you a choice.
 
-## Specific Recommendations
+Writing the docs is extremely useful for project's users and developers, and grows your karma :-)
 
-* Documentation should make sense when you read it through from beginning to end. If you add new content, try to place it where the necessary concepts have already been explained.
-* If a documentation section consists of many similar items, like functions or operators, try to order them from more generic (usable by a wide audience) to more specific (for specific use cases or application types). If several items are intended to be mostly used together, group them together in the documentation.
-* Try to avoid slang. Use the most common and specific terms possible for everything. If some terms are used as synonyms, state this explicitly.
-* All descriptions of functionality should be accompanied by examples. Basic examples are acceptable, but real world examples are welcome, too.
-* Sensitive topics like politics, religion, race, and so on are strictly prohibited in documentation, examples, comments, and code.
-* Proofread your text before publishing. Look for typos, missing punctuation, or repetitions that could be avoided.
-* Try to avoid addressing the reader directly, although this is not strictly prohibited.
+**Contents**
 
-# How to Add a New Language
+- [What is the ClickHouse Documentation](#clickhouse-docs)
+- [How to Contribute to ClickHouse Documentation](#how-to-contribute)
+    - [Markdown Dialect Cheatsheet](#markdown-cheatsheet)
+    - [Adding a New File](#adding-a-new-file)
+    - [Adding a New Language](#adding-a-new-language)
+-  [How to Write Content for ClickHouse Documentation](#what-to-write)
+    - [Documentation for Different Audience](#target-audience)
+    - [Common Recommendations](#common-recommendations)
+    - [Description Templates](#templates)
+- [How to Build Documentation](#how-to-build-docs)
+
+
+<a name="clickhouse-docs"/>
+
+## What is the ClickHouse Documentation
```

Comment:
I give up for now... it doesn't look lightweight

Reply:
There is no much text further :-).